### ISSUE TITLE
Image cannot be downloaded correctly

### ISSUE THREAD

**Issue by qianxuyidian-2018** [Tencent/tencent-ml-images#8]
when i ran the script to download image from URL that listed in train_url_tiny.txt, i got the bad image.
![im_0](https://user-images.githubusercontent.com/39649910/47253692-3f281680-d489-11e8-9398-78dc3ef55639.jpeg)
when i copied the URL to web browser，i can see the correct image。
Do you know the cause of the problem?

**Comment by Gwinki** [comment]
you should add  “url = url.split(' ')[0]” after “url = sp[0]”. I have printed the url and  find that the output are not only the image's url.

**Comment by gongwk** [comment]
maybe in linux space means ‘\t’, but in windows that just means a space

**Comment by wubaoyuan** [comment]
@gongwk @qianxuyidian-2018 Thanks for your feedbacks. The demo wroks in my server. I will check it again.  

Actually, we have realized the invalid URLs and the downloading speed are the main barrier to utilize our data. We are trying to find an approach to access the data more easily.

**Comment by qianxuyidian-2018** [comment]
@gongwk you are right,I ran the script in windows.It is ok when i changed "\t" to " ".thank you!
@wubaoyuan thanks,i will try the new script.
